src/Shared/HttpSys/RequestProcessing/NativeRequestContext.cs

```diff
@@ -221,11 +223,11 @@ public virtual void Dispose()
     internal string? GetVerb()
     {
         var verb = NativeRequest->Verb;
-        if (verb > HttpApiTypes.HTTP_VERB.HttpVerbUnknown && verb < HttpApiTypes.HTTP_VERB.HttpVerbMaximum)
+        if (verb > HTTP_VERB.HttpVerbUnknown && verb < HTTP_VERB.HttpVerbMaximum)
```

Comment:
This is sketchy, can we at a minimum debug assert that `HttpVerbs` has `HTTP_VERB.HttpVerbMaximum` number of elements.